cmake/packages.cmake

```diff
@@ -0,0 +1,270 @@
+
```

Comment:
The name might be a bit too generic and easy to be confused with `config/packages.cmake`.